src/components/sidebar/components/sidebar-headings/index.tsx

```diff
@@ -39,4 +46,22 @@ const SidebarSectionHeading = ({ text }: SidebarSectionHeadingProps) => {
 	return <h3 className={s.root}>{text}</h3>
 }
 
-export { SidebarSectionHeading, SidebarTitleHeading }
+const SidebarSectionBrandedHeading = ({
+	text,
+	theme,
+}: SidebarSectionBrandedHeadingProps) => {
+	return (
+		<h3 className={classNames(s.brandedRoot, s[`theme-${theme}`])}>
```

Comment:
WDYT about making this `<h3 className={classNames(s.brandedRoot, s[theme])}>` and updating the css classes from `theme-<product>` to `<product>?